### PR TITLE
file_tile + tag_tile + file_upload_page

### PR DESCRIPTION
**Pull request by shellyannissa**

### CODE REVIEW

File: lib/services/enums/file_type.dart
Comment (shellyannissa) [inline lib/services/enums/file_type.dart:1]: kindly review this @dhwanish-3

**Review by shellyannissa — COMMENTED** [review]
New page with widgets 
- searchBar
- FileTile
- TagTile
- MonthDivider

File: lib/services/enums/file_type.dart
Comment (dhwanish-3) [reply lib/services/enums/file_type.dart:1]: I approve

**Review by dhwanish-3 — APPROVED** [review]
Approved

**Pull request merged**